src/python/WMCore/Services/TagCollector/TagCollector.py

```diff
@@ -5,31 +5,34 @@
 from collections import defaultdict
 from WMCore.Services.Service import Service
 from WMCore.Services.TagCollector.XMLUtils import xml_parser
-
+from urlparse import urlparse
 
 class TagCollector(Service):
     """
     Class which provides interface to CMS TagCollector web-service.
     Provides non-deprecated CMSSW releases in all their ScramArchs (not only prod)
     """
 
-    def __init__(self, url=None, logger=None, **kwargs):
+    def __init__(self, url=None, logger=None, configDict=None, **kwargs):
         """
         responseType will be either xml or json
         """
         defaultURL = "https://cmssdt.cern.ch/SDT/cgi-bin/ReleasesXML"
+        url = url or defaultURL
+        parsedUrl = urlparse(url)
```

Comment:
For future reference, I'm not sure we really need to have this url parsing for the cache name.